src/main/java/org/example/HibernateSessionFactoryUtil.java

```diff
@@ -0,0 +1,33 @@
+package org.example;
+
+
+import org.example.model.Employee;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+
+import java.lang.module.Configuration;
```

Comment:
Неправильный импорт.
Должен быть такой:
```suggestion
import org.hibernate.cfg.Configuration;
```